src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- It looks like you're project doesn't have tests, I kindly invite you to add some of them. As much as you need to cover at least the unit test.
![image](https://user-images.githubusercontent.com/20563252/209450286-608d9a62-413f-4458-885d-e4ed0455315a.png)